Cargo.toml

```diff
@@ -17,6 +17,7 @@ rustdoc-args = ["--cfg=doc_cfg"]
 [dependencies]
 bytes = "1.3.0"
 futures-io-03 = { package = "futures-io", version = "0.3.25", optional = true }
+once_cell = "1.17"
```

Comment:
Probably going to have to reduce the version requirement here because we advertise support against pretty old versions of rustc.